Makefile

```diff
@@ -0,0 +1,30 @@
+SHELL := /bin/bash
+
+.PHONY: .rocks
+.rocks: graphql-scm-1.rockspec Makefile
+		tarantoolctl rocks make
+		tarantoolctl rocks install luatest 0.5.5
+		tarantoolctl rocks install luacov 0.13.0
+		tarantoolctl rocks install luacheck 0.26.0
+
+.PHONY: lint
+lint:
+		if [ ! -d ".rocks" ]; then make .rocks; fi
```

Comment:
That's a bit strange. You can make `.rocks` non-phony target to achieve the same behaviour. But, it seems, you don't want, because it may miss some dependencies (say, after manual `tarantoolctl rocks make`). That's minor, though.

Reply:
I borrowed it from original PR, but I think the motivation was the same as your assumption.

Reply:
I would do it so:

```
.rocks/bin/luacheck
        tarantoolctl rocks install luacheck <ver>

.rocks/bin/luatest
        tarantoolctl rocks install luatest <ver>

.rocks/bin/luacov
        tarantoolctl rocks install luacov <ver>

.PHONY: dependencies
dependencies:
        tarantoolctl rocks make

.PNOHY: dependencies-test
depdendencies-test: depdendencies .rocks/bin/luacheck .rocks/bin/luatest .rocks/bin/luacov
        true
```

But simpler is better and if everything typically works okay, I propose to don't make things more complex.